infra-templates/ovh-dns/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+ovh:
+  image: rancher/external-dns:v0.8.0
```

Comment:
Image released including ovh provider is rancher/external-dns:v0.7.10, could you please update it?? 
```
-  image: rancher/external-dns:v0.8.0
+  image: rancher/external-dns:v0.7.10
```